lib/bloc/redeemBloc/redeem_state.dart

```diff
@@ -0,0 +1,27 @@
+part of 'redeem_bloc.dart';
+
+class RedeemState extends Equatable {
+  final bool sucessful;
+  factory RedeemState.initial() {
+    return RedeemState(sucessful: false);
+  }
+
+  RedeemState({
+    required this.sucessful,
+  });
+
+  @override
+  List<Object> get props {
+    return [
+      sucessful,
+    ];
+  }
+
+  RedeemState copyWith({
+    bool? sucessful = false,
+  }) {
+    return RedeemState(
+      sucessful: sucessful ?? this.sucessful,
```

Comment:
Not a bloc expert in this case, but should we dedicate a child of RedeemState per "State".
I mean if we would want to consider for example an error in the RedeemState then having both sucessful and the error in the same class would cause a bit of confusion in future readability and testing.

Again, I am no bloc expert and I also want to clarify this a bit for myself.